locales/ja-JP/src/structure.ts

```diff
@@ -102,9 +103,10 @@ export default removeUndefinedLocaleResources({
   'banners.permission-check-banner.missing-permission_update_other':
     'あなたの役割<Roles/>はこのドキュメントを更新する権限がありません。',
   /** The pending text for the request permission button that appears for viewer roles */
-  'banners.permission-check-banner.request-permission-button.sent': undefined, // 'Editor request sent'
+  'banners.permission-check-banner.request-permission-button.sent':
+    '編集リクエストが送信されました',
```

Comment:
Is this like someone requests for editor? or just edit the document?

Reply:
It's when a user who has the "viewer" role (eg is only able to view a document, not edit it) clicks "Request edit access" - this is the confirmation message shown after this occured.